FormalConjectures/ErdosProblems/203.lean

```diff
@@ -0,0 +1,32 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Erdős Problem 203
+
+*Reference:* [erdosproblems.com/203](https://www.erdosproblems.com/203)
+-/
+
+/--
+Is there an integer m with (m, 6) = 1 such that none of 2^k * 3^ℓ * m + 1 are prime,
+for any k, ℓ ≥ 0?
+-/
+@[category research open, AMS 5]
+theorem erdos_203 : ∃ m : ℕ, Nat.Coprime m 6 ∧ (∀ k l : ℕ, ¬ Nat.Prime (2^k * 3^l * m + 1))
+  ↔ answer(sorry) := by
+sorry
```

Comment:
```suggestion
@[category research open, AMS 5]
theorem erdos_203 : (∃ m, m.Coprime 6 ∧ ∀ k l, ¬ (2^k * 3^l * m + 1).Prime) ↔ answer(sorry) := by
  sorry
```

One needs parenthesis around the entire statement so that `↔  answer(sorry)` works correctly. The other changes are more cosmetic.